src/__tests__/components/NavBar/ShoppingCart.test.tsx

```diff
@@ -1,13 +1,14 @@
 import { render, screen } from "@testing-library/react";
 import ShoppingCart from "@/components/NavBar/ShoppingCart";
+import renderWithMockedProvider from "../../utils";
 
 describe("ShoppingCart", () => {
   const props1 = {
     quantity: 5,
     loginState: true,
   };
   it("Should render the quantity '5' and circle element successfully", () => {
-    const { getByText } = render(
+    const { getByText } = renderWithMockedProvider(
```

Comment:
Could you please explain the reason for using mock data? Thank you!

Reply:
We used the renderWithMockedProvider since we initially tried to test the ShoppingCart icon on navbar interaction. The cart icon click event needs to involve the redux store, thus, we used the mock provider.  After the discussion with Tato and Blue, the test I mentioned should not be involved in this stage. Using the simple "render" is enough in those tests.